expandybird/expansion/expansion.py

```diff
@@ -232,7 +232,7 @@ def ExpandTemplate(resource, imports, env, validate_schema=False):
     except schema_validation.ValidationErrors as e:
       raise ExpansionError(resource['name'], e.message)
 
-  if path.endswith('jinja'):
+  if path.endswith('jinja') or path.endswith('yaml'):
```

Comment:
Because the helm files are named yaml, so I'd rather not change their names on the fly.